app/models/service_interaction.rb

```diff
@@ -0,0 +1,2 @@
+class ServiceInteraction < ActiveRecord::Base
+end
```

Comment:
The identifiers and labels for both services and interactions should be unique, so we should validate them for both presence and uniqueness at least 🎁 ❄️